recipe/meta.yaml

```
@@ -9,12 +9,16 @@ source:
   - url: https://github.com/colmap/{{ name }}/archive/v{{ version }}.tar.gz
     sha256: cead12638a9746143143ba29a5cb6131a34d6e9555ade9c9d1b79b8eacfbf9cf
     patches:
+      - use-cmake-args.patch
       - use-conda-pybind11.patch
       - fix-win-build.patch
+      - fix-find-package-cuda.patch
 
 build:
-  number: 2
-  script: {{ PYTHON }} -m pip install . -vv
+  number: 3
+  script:
+    - export CMAKE_ARGS="${CMAKE_ARGS} -DCUDAToolkit_ROOT=/usr/local/cuda/targets/sbsa-linux"  # # [cuda_compiler_version == "11.*" and aarch64]
```

Adding this argument only does something if the CMakeLists are calling find_pacakage(CUDAToolkit)

---

Also linter is raising an error related to this. Think it would need to be

```suggestion
    - export CMAKE_ARGS="${CMAKE_ARGS} -DCUDAToolkit_ROOT=/usr/local/cuda/targets/sbsa-linux"  # [cuda_compiler_version == "11.*" and aarch64]
```